### PR TITLE
Add filter options endpoint

### PR DESCRIPTION
**Pull request by atatmaja**

### CODE REVIEW

File: app/utils/get_filter_options.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+from sqlalchemy import distinct, func
+from itertools import groupby
+from functools import reduce
+
+def get_all_filter_options():
+    with db_session() as session:
+        # List of fields in AirtableSource to query
+        airtable_field_strings = ['source_type', 'study_status', 'overall_risk_of_bias', 'sex', 'estimate_grade']
+        options = {}
+        for field_string in airtable_field_strings:
```
Comment (simonarocco) [inline app/utils/get_filter_options.py:14]: why don't you just combine this for loop and the one below into one and add the airtable_field_strings to the other_fields dict and just set their table to AirtableSource? not sure if there is a need to have them separate

**Review by simonarocco — COMMENTED** [review]
lgtm just see my one suggestion

File: app/utils/get_filter_options.py (hunk 1 above)
Comment (atatmaja) [reply app/utils/get_filter_options.py:14]: yeah fair point

Comment (atatmaja) [reply app/utils/get_filter_options.py:14]: actually I'm going to push back on this. for all the other_fields, there is only one field per table that we need to scrape, and that field's name must be changed. that's a bit different from the fields we need to scrape from airtable_source. 

could probably refactor into one loop but I think this is pretty readable and will not really incur much of a perf diff.

**Pull request merged**